src/powerquery-language-services/inspection/scope/scopeUtils.ts

```diff
@@ -58,7 +58,7 @@ export function findScopeItemByLiteral(
     nodeScope: NodeScope | undefined,
     literalString: string,
 ): TScopeItem | undefined {
-    return nodeScope?.get(`@${literalString}`) ?? nodeScope?.get(literalString);
+    return nodeScope?.get(literalString);
```

Comment:
Don't feel bad about needing a hotfix, I believe I was the one who told you to change it to this in the first place.

Reply:
🤕